x-pack/plugins/data_enhanced/public/search/sessions_mgmt/lib/date_string.ts

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Can we move this function into the `common` folder? I imagine it'd be useful for validating both client/server.